spec/modules/helpers.rb

```diff
@@ -0,0 +1,88 @@
+module SessionHelpers
+  def create_user(role)
+    User.create(
+      login: OmniAuth.config.mock_auth[:google_oauth2][:info][:name],
+      provider: OmniAuth.config.mock_auth[:google_oauth2][:provider],
+      uid: OmniAuth.config.mock_auth[:google_oauth2][:uid],
+      auth_role: role,
+    )
+  end
+
+  def login_as_user
+    create_user("user")
+    visit root_path
+    click_on "Login"
+    expect(page).to have_text("Logged in!")
+  end
+  
+  def login_as_moderator
+    create_user("moderator")
+    visit root_path
+    find_button "Login"
+    click_on "Login"
+    expect(page).to have_text("Logged in!")
+  end
+
+  def login_as_admin
+    create_user("admin")
+    visit root_path
+    click_on "Login"
+    expect(page).to have_text("Logged in!")
+  end
```

Comment:
What do you think about, instead of having 3 methods, this be one method that accepts a parameter of the role?  That way you can call `login_as(:user)` or `login_as(:moderator)`.

Reply:
Done => [0ce100f](https://github.com/Brian-Demon/conclave/pull/30/commits/0ce100fa7ea6d739793f55072994354c646bb65c)